.github/workflows/cli.yml

```diff
@@ -10,8 +10,6 @@ on:
 env:
   golang-version: 1.18.1
   golangci-lint-version: v1.45.2
-  # cert-manager-version needs to be also updated in cli/pkg/otel-utilities.go
-  cert-manager-version: v1.6.1
```

Comment:
This is moved into Makefile. Renovate configuration is also adjusted for this change.